setup.py

```diff
@@ -89,6 +89,9 @@
         'Framework :: Django',
         'Framework :: Django :: 1.11',
         'Framework :: Django :: 2.0',
+        'Framework :: Wagtail',
+        'Framework :: Wagtail :: 1',
+        'Framework :: Wagtail :: 2',
```

Comment:
I'm not sure about listing both `Framework :: Wagtail :: 1` and `Framework :: Wagtail :: 2` as one of the two will be wrong for every release going forward (i.e. release for `2.1.x` or `2.2` shouldn't list `Framework :: Wagtail :: 1` and LTS releases for `1.12` and `1.13` shouldn't list `Framework :: Wagtail :: 2`).

Reply:
That was my initial thought as well - you'll see I added `Wagtail :: 1` in a second commit. It was pointed out to me that if people are looking through PyPI, we will want Wagtail listed within the `Wagtail 1` sub-category, so they can wayfind to the latest version. What do you think?

Reply:
To me classifiers were supposed to be used by libraries to declare compatibility with a given Wagtail version, not as a mechanism to find Wagtail in searches.

The [classifier's documentation](https://pypi.org/classifiers/) is a bit ambiguous though:

> Each project's maintainers provide PyPI with a list of "trove classifiers" to categorize each release, describing who it's for, what systems it can run on, and how mature it is.
>
> These standardized classifiers can then be used by community members to find projects based on their desired criteria.

The first sentence kinda describe the project requirements in a way when the second sentence indeed talk about search.

FWIW, [Django does not declare its own classifiers](https://github.com/django/django/blob/5b733171813f8ddc7af84abe79f2646204b9c6ca/setup.py#L92-L109).

So in the end I don't know what to think.

Reply:
@loicteixeira Django does declare its own classifier: https://github.com/django/django/blob/5b733171813f8ddc7af84abe79f2646204b9c6ca/setup.py#L95

I think it makes sense to follow the same approach without the version number – Wagtail will be findable, but won't try to convey any kind of compatibility with any specific version of itself.

Reply:
I don't know how I missed this one sorry :(